terraform/deployments/govuk-publishing-infrastructure/csvs_s3.tf

```
@@ -0,0 +1,45 @@
+resource "aws_iam_policy" "csvs_bucket_polices" {
```

nit: I only just noticed the typo in the name here 😅 Now that I think about the name, how about:
```suggestion
resource "aws_iam_policy" "write_csvs_buckets" {
```

---

haha, yeah i'd not even noticed that, going to blame the dyslexia though